widget/Toaster.js

```diff
@@ -79,7 +81,8 @@ define([
 			this.ownerDocument.body.appendChild(this.domNode);
 
 			if(this.messageTopic){
-				connect.subscribe(this.messageTopic, this, "_handleMessage");
+				// TODO: wrap in a call to this.own
+				this.own(topic.subscribe(this.messageTopic, lang.hitch(this, "_handleMessage")));
```

Comment:
As I said before, you can get rid of the TODO comment here, and also it's slightly better to just call `this.subscribe()` instead of `this.own(topic.subscribe(...`